infra-templates/k8s/37/rancher-compose.yml

```diff
@@ -220,6 +220,10 @@ kubernetes:
         sans:
         - "IP:${KUBEAPI_CLUSTER_IP}"
         - "kubernetes.default.svc.cluster.local"
+        - "kubernetes.default.svc.cluster"
+        - "kubernetes.default.svc"
+        - "kubernetes.default"
```

Comment:
these 2 tags - like 225 and 226- are already being added by cattle https://github.com/rancher/cattle/blob/v1.6/code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/service/impl/ServiceDiscoveryApiServiceImpl.java#L855

Reply:
so remove `kubernetes.default`and `kubernetes` from the list